config/locales/server.en.yml

```diff
@@ -2898,6 +2898,7 @@ en:
     not_approved: "Your account hasn't been approved yet. You will be notified by email when you are ready to log in."
     incorrect_username_email_or_password: "Incorrect username, email or password"
     incorrect_password: "Incorrect password"
+    password_policy_requirement_not_met: "Your current password does not meet new password policy requirements for this site. Please check your email for password reset instructions."
```

Comment:
Wording should be "Your old password has expired. Please check your email for password reset instructions." per internal topic. The one here would be used when the password does not match up with site setting requirements.

Reply:
Also key would be `password_expired`